example/src/components/atoms/TestStateDisplay/TestStateDisplay.tsx

```diff
@@ -21,7 +21,7 @@ export function TestStateDisplay({ label, state, value }: TestStateDisplayProps)
     <Container>
       <Header>
         <Label>{label}</Label>
-        <AppText>{marks[state]}</AppText>
+        {state && <AppText>{marks[state]}</AppText>}
```

Comment:
It's better to use a terniary in views, so:
```
{state ? <AppText>{marks[state]}</AppText> : null}
```

If a boolean false would be passed to `state` it would now visualize `False`.

Reply:
Are you sure? As far as I know, visualizing False was removed a few years ago. When conditional rendering gets false it just not rendering (renders nothing in this place). The problem will only appear when you use the number 0 as a condition. The component allows you to pass only 'DONE' | 'WAITING' | 'ERROR' | 'IN_PROGRESS' values in this place. As a result, in this case, adding `: null` does not help at all, but prolongs the code. To be fully safe, all we need to do is parse the condition to boolean. 
eg.
<img width="1144" alt="Screenshot 2023-10-11 at 11 02 17" src="https://github.com/dotintent/react-native-ble-plx/assets/143402866/c837b248-83ab-4a5b-89da-426e5b871fa1">